tests/integration/acp/add_policy/with_empty_args_test.go

```diff
@@ -42,8 +42,6 @@ func TestACP_AddPolicy_EmptyPolicyCreator_Error(t *testing.T) {
 
 		Actions: []any{
 			testUtils.AddPolicy{
-				Identity: "",
```

Comment:
todo: I don't like these changes, the tests should explicitly show the identity being empty.

The previous tests should stay as much as possible in my opinion. I think what you want to test in this PR and what the previous tests test are 2 fundamentally separate things.

The previous tests assert that the RAW identity (without being wrapped in auth token) work. Which should continue behaving like they did.

You should introduce a new Variable called `IdentityWithAuth` within the respective `testUtils.XYZ` util functions where you want to test the authentication, then copy the tests you think are relavent from `tests/integration/acp` and add them to a dir called `test/integration/acp/auth`. This way we don't mix the testing of authentication and the acp logic.

So al/mostl the previous tests should be reverted and unchanged in this PR and new one introduced with using `IdentityWithAuth` instead of `Identity` (you this way also don't have to re-test  all acp cases, but only the authentication functionality.

Reply:
> The previous tests assert that the RAW identity (without being wrapped in auth token) work. Which should continue behaving like they did.

This is related to one of my questions I think.  We might want to chat as a team as to whether we want to support unauthenticated identities in the future.  The jwt token is also required for SourceHub as far as I can tell, and the usefulness of local-ACP is debated atm.

Reply:
When I said 

> Which should continue behaving like they did.

I meant the access control part should behave normally as it did, sorry. Not that we should keep non-auth identity functionality.

In the comment above I am suggesting to not mix the "authentication" testing with the "access control" testing, and keep them separate.

As for weather we want unauthenticated identities, I really can't think of a good reason why we would want to keep that if we have auth implemented.

As for:

> the usefulness of local-ACP is debated atm

I don't see how it's debated haha, IMO is really useful for anyone who wants to use defradb like a classic/normal use case (i.e. without a sourcehub-like instance). We can test in standup though, maybe I am missing something.

Reply:
> In the comment above I am suggesting to not mix the "authentication" testing with the "access control" testing, and keep them separate.

How would you keep them separate without permitting auth-free identities?

Reply:
> The previous tests assert that the RAW identity (without being wrapped in auth token) work. Which should continue behaving like they did.

The Go client doesn't create auth tokens. Only the HTTP / CLI clients use auth tokens.

The change here was made because the new identity requires a valid public or private key and an empty string is not a valid identity.